charts/fybrik/values.yaml

```diff
@@ -14,10 +14,15 @@ clusterScoped: true
 
 # The namespace where Fybrik deploys data path components (modules)
 modulesNamespace:
-  # The Fybrik namespace name, if not defined, default "fybrik-blueprints" is used
+# The Fybrik namespace name, if not defined, default "fybrik-blueprints" is used.
+# If `app.fybrik.io/modules-namespace` label is set in FybrikApplication resource then
+# it takes precedence over other values defined. In this case, it is assumed that Fybrik
+# is not responsible to create this namespace and remove it when Fybrik is uninstalled.
+# Moreover, it is a user responsibility to give Fybrik the permissions to deploy modules in this namespace.
   name: ""
   # if it is true, Fybrik, during its deployment, creates this namespace and removes it when Fybrik is uninstalled.
   # Otherwise, it is a user responsibility to create this namespace, before Fybrik deployment.
+  # Relevant only for namespace defined in .Values.modulesNamespace field.
```

Comment:
Fybrik also provides RBAC permissions to edit/view resources in this namespace, if set to true.
Perhaps it's better to rephrase to something like this:
`If set to true, Fybrik manages creation and deletion of the namespace above as well as providing necessary RBAC permissions to orchestrate modules in this namespace. Otherwise, it is a user responsibility to create this namespace with the appropriate permissions prior to Fybrik deployment.`

Reply:
Changed. Thanks